### ISSUE TITLE
Improve contrast for the "Quick Fix" button

### ISSUE THREAD

**Issue by mlewand** [cksource/ckeditor-plugin-a11ychecker#162]
Based on our partner's feedback:

> Colour Contrast fails for Text & Background colour on Quick Fix buttons fails AVT Checkpoint 1.4.3 Contrast(Minimum) . Please see below screenshot:

![06363347](https://cloud.githubusercontent.com/assets/5353898/13007951/bc8b5480-d193-11e5-8b4b-8720b5b45a67.png)

**Comment by mlewand** [comment]
We'll support contrast changes with new skin introduced in CKEditor 4.6.0, moono-lisa. Problem has been resolved with #228.
